cmd/crio/wipe.go

```diff
@@ -51,15 +51,15 @@ func crioWipe(c *cli.Context) error {
 		}
 	}
 
-	// we should not wipe images without wiping containers
-	if shouldWipeImages && !shouldWipeContainers {
-		logrus.Infof("version file in tmpfs out of date from version file in persistent directory. Triggering container wipe anyway")
-		shouldWipeContainers = true
-	}
-
 	// if we should not wipe, exit with no error
-	// since we won't wipe images without wiping containers, we can just check the one condition
 	if !shouldWipeContainers {
+		// we should not wipe images without wiping containers
+		// in a future release, we should wipe both container and images if only shouldWipeImages is true.
+		// However, now, we cannot expect users to have version-file-persist after having upgraded
+		// to this version. Skip the wipe, for now, and log about it.
+		if shouldWipeImages {
+			logrus.Infof("legacy version-file path found, but new version-file-persist path not. Skipping wipe")
+		}
```

Comment:
Do we have a "TODO:" tag that we use to make sure we clean stuff up?  And/or is there an issue spun up for this that we could reference here?

Reply:
I intend on cherry-picking this version into 1.18, then writing the new version in master. I am leaving myself in charge of remembering :smile: